Frontend/BetterSteam/src/components/GameBox.vue

```diff
@@ -9,45 +9,58 @@
       <div class="column is-two-third gameinfo">
         <p class="gametitle">{{ Game.name }}</p>
         <p class="gamecompany">{{ Game.company }}</p>
+        <button
+          class="betterSteamButton--wishlist"
+          @click="isWishlisted = !isWishlisted"
+          v-if="isAllowedToFavAndWish"
+        >
+          <span class="icon" v-if="wishlistable">
+            <i :class="wishGameClass" class="fa-bookmark fa-xl" />
+          </span>
+        </button>
         <span class="gameprice">{{ `${Game.price.toFixed(2)}€` }} </span>
         <p class="gamedate">{{ formattedDate }}</p>
       </div>
     </div>
     <button
+      v-if="isAllowedToFavAndWish"
       class="betterSteamButton--favorite"
-      @click="isFavorited = !isFavorited"
+      @click="handleFavorite"
     >
-      <span class="icon" v-if="Favoriteable">
-        <i :class="favGameClass" class="fa-star fa-lg" />
+      <!-- we need Backend functionality later for this -->
+      <span class="icon" v-if="favoriteable">
+        <i :class="favGameClass" class="fa-star fa-xl" />
       </span>
     </button>
   </div>
 </template>
 <script lang="ts">
-import { Vue } from "./";
+import { Vue, Cookies } from "./";
 export default Vue.extend({
   name: "GameBox",
   props: {
     Game: { type: Object, default: undefined },
     enableHover: { type: Boolean, default: true },
-    Favoriteable: { type: Boolean, default: true },
+    favoriteable: { type: Boolean, default: true },
```

Comment:
Kleiner Exkurs (5min.) über kombinierbare Booleans/Enums mit etwas moderner Pop-Culture:
[Nicht Yandere-Simulator](https://youtu.be/LleJbZ3FOPU?t=2459)